solution/ui/regulations/eregs-vite/src/views/PolicyRepository.vue

```diff
@@ -68,16 +84,20 @@ const getPartsLastUpdated = async () => {
     }
 };
 
+// policyDocList fetch for policy document list
 const policyDocList = ref({
     results: [],
     loading: true,
 });
 
-const getDocList = async () => {
+const getDocList = async (requestParams = "") => {
+    policyDocList.value.loading = true;
+
     try {
         policyDocList.value.results = await getPolicyDocList({
             apiUrl: props.apiUrl,
             cacheResponse: !props.isAuthenticated,
+            requestParams,
         });
     } catch (error) {
```

Comment:
would it make sense to have policyDocList.value.loading = false on the exception.  Its no longer loading but an error

Reply:
the `finally` block on line 104 runs no matter what, even if an exception is thrown.  So it would flip `policyDocList.value.loading` to false even with an error.

Reply:
Didnt see the finally.  All good then